cmd/main.go

```diff
@@ -52,7 +52,7 @@ type Data struct {
 	// Scope options.
 	Scope Scope `json:"scope"`
 	// Rules options.
-	Rules *Rules `json:"rules"`
```

Comment:
**hint:** There is not value (only complexity) for _Rules_ to be a pointer.
Changing for simplicity and consistency.